rust/worker/src/execution/operator.rs

```diff
@@ -65,6 +65,7 @@ where
 #[derive(Debug)]
 pub(super) struct TaskResult<Output, Error> {
     result: Result<Output, TaskError<Error>>,
+    #[allow(dead_code)]
```

Comment:
Rather than allowing dead code and spreading the dead code, can we clean it up?

Reply:
I left it here because I think this field might be useful in the future (e.g. if an orchestrator needs to trace / persist the result)

For context, it was used by the old hnsw orchestrator where it needs to track multiple concurrent flows of compute. It is not required now because the new orchestrators do not have this need.

Reply:
At a closer look it is used in some unit tests so probably we should not remove this.